Ardronev2 support

clero opened the pull request:
This PR add the AR Drone v2 support to AR.Pwn project.
It has been updated with values tested on another project.

_I have no access to drones for now so those patchs have not been tested._

The vision project has also been updated, please check IDE compilation as I have built through command line.

clero committed:
Rename buffers for AR-Drone v2 support

Buffers for vertical and horizontal camera are not the same
for AR-Drone v1 and v2.
This patch renames buffers' name to CAM1 and CAM2 to avoid
hardware device association.

Signed-off-by: Jules Clero <clero@etud.insa-toulouse.fr>

JeremyRand commented:
Thanks for the PR.  I'm travelling at the moment, but will test this as soon as I can.

clero committed:
hook: Add support of ArDrone v2

The file device and the image size differs between ArDrone v1
and v2.
This patch introduces different camera definition. ArDrone v1
definition is used by default. The ArDrone v2 one can be used
by compiling with -DDRONEV2 flag.

Signed-off-by: Jules Clero <clero@etud.insa-toulouse.fr>

clero committed:
vision: Use image size defined in hook

Signed-off-by: Jules Clero <clero@etud.insa-toulouse.fr>
